cristin-import/src/main/java/no/unit/nva/cristin/mapper/nva/exceptions/ExceptionHandling.java

```diff
@@ -33,9 +33,6 @@ public static RuntimeException castToCorrectRuntimeException(Exception exception
         if (exception instanceof DuplicateDoiException) {
             return (DuplicateDoiException) exception;
         }
-        if (exception instanceof InvalidArchiveException) {
-            return (InvalidArchiveException) exception;
-        }
```

Comment:
Removing InvalidArchive exception. Instead of not importing publications with invalid uri's, we import them without these uri's.